pkg/pillar/zedcloud/deferred.go

```diff
@@ -130,6 +129,10 @@ func (ctx *DeferredContext) handleDeferred(log *base.LogObject, event time.Time,
 				log.Functionf("handleDeferred: for %s failed %s",
 					key, err)
 				exit = true
+			} else if result != types.SenderStatusNone {
+				log.Functionf("handleDeferred: for %s received unexpected status %d",
+					key, result)
+				exit = true
```

Comment:
Should we try with just this addition and see if the AKID failures go away?

Reply:
Done. Also added some logging around certificates.